ThingIFSDK/ThingIFSDK/Command.swift

```diff
@@ -100,7 +86,7 @@ open class Command: NSObject, NSCoding {
      - Parameter alias: Alias to get action.
      - Returns Array of `AliasAction`.
      */
-    open func getAction(_ alias: String) -> [AliasAction] {
```

Comment:
I think this method name `getAction`.

This is a method corresponding to following java method:
```java
    public <T extends Action> List<AliasAction<T>> getAction(
            @NonNull String alias,
            @NonNull Class<T> clsOfT)
```

What do you think?